app/routes/Event.tsx

```diff
@@ -0,0 +1,12 @@
+import Calendar from '~/components/Calendar.client';
+
+// This will changed - TODO
```

Comment:
Changed to what? can we add a small description? or may create a to-do ticket and update the comment with that link? so we have a track for it?